deploy/examples/toolbox-operator-image.yaml

```
@@ -0,0 +1,136 @@
+#################################################################################################################
+# Define the toolbox that will run with the Rook operator image.
+
+# For example
+#   kubectl create -f toolbox-rook-image.yaml
+#################################################################################################################
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: rook-ceph-tools-rook-image
```

Let's match the name and other properties with the file name
```suggestion
  name: rook-ceph-tools-operator-image
```